src/coreclr/vm/datadescriptor/datadescriptor.inc

```diff
@@ -1144,6 +1144,9 @@ CDAC_GLOBAL_POINTER(MethodDescSizeTable, &MethodDesc::s_ClassificationSizeTable)
 CDAC_GLOBAL_POINTER(GCLowestAddress, &g_lowest_address)
 CDAC_GLOBAL_POINTER(GCHighestAddress, &g_highest_address)
 
+
+// It is important for the subdescriptor pointers to be the last pointers in the global structure
+// enummem.cpp relies on this order to find and enumerate the subdescriptors
 CDAC_GLOBAL_SUB_DESCRIPTOR(GC, &(g_gc_dac_vars.gc_descriptor))
```

Comment:
I'd also add a comment about the hard-coded number of sub-descriptors so that if anyone adds another one they know to update the enum mem code too.